src/components/PageContainer/component.js

```diff
@@ -252,7 +252,11 @@ class PageContainer extends Component<DefaultProps, Props, void> {
         return <PageLoading />
       }
       else if (Layout) {
-        return <Layout ref={ this.saveContentRef } { ...page } />
+        return (
+          <div ref={ this.saveContentRef }>
```

Comment:
This is not an option. I recently made a change to remove this div. I know that ref cannot be used on stateless component, but I don't want a div to be added neither.